RELEASE_NOTES.md

```diff
@@ -31,6 +31,7 @@ other improvements. See the ixmp release notes for further details.
 - [#154](https://github.com/iiasa/message_ix/pull/154): Enable documentation build on ReadTheDocs.
 - [#138](https://github.com/iiasa/message_ix/pull/138): Update documentation and tutorials.
 - [#131](https://github.com/iiasa/message_ix/pull/131): Update clone function argument `scen` to `scenario` with planned deprecation of the former.
+- [#232](https://github.com/iiasa/message_ix/pull/232): Adding a Westeros tutorial for modelling seasonality and updating the existing ones
```

Comment:
This should be at the top, under the "Next release" section, rather than the section for the already-released v1.2.0.

Reply:
Thanks @khaeru for noticing this. I corrected.